### DISCUSSION TITLE
Support for long-press on scene buttons iCasa Pulse 8S (ICZB-KPD18S)

### DISCUSSION THREAD

**Discussion by GingerAdonis** [Koenkk/zigbee2mqtt#14873] [Devices]
Hi. I have an iCasa Pulse 8S (ICZB-KPD18S), which works great with zigbee2mqtt.

However, I'm missing the possibility for an action event whenever one of the scene buttons gets pressed for a longer amount of time. Is that something that needs to be added in zigbee2mqtt, or is it necessary to create a workaround?

Here's a Home Assistant ZHA blueprint with long press support for reference https://community.home-assistant.io/t/zha-icasa-pulse-s8-keypad-remote/267243

Thanks in advance. :)

**Comment by DeanR97** [comment]
I have implemented this in this pull request 
https://github.com/Koenkk/zigbee-herdsman-converters/pull/6222

this should be in the next update :)
